Sources/XCTest/Private/PerformanceMeter.swift

```diff
@@ -11,7 +11,7 @@
 //  Measures the performance of a block of code and reports the results.
 //
 
-#if os(Linux) || os(FreeBSD)
+#if USE_OBJC_FOUNDATION || os(Linux) || os(FreeBSD)
```

Comment:
OK, this is very close to what I had in mind. My only suggestion would be to remove the `|| os(Linux) || os(FreeBSD)` here. Instead, have the build script, `build_script.py`, define `USE_OBJC_FOUNDATION=1` when compiling for a non-Darwin target.

Of course, it's strange that `USE_OBJC_FOUNDATION` should be defined when building for Linux, so I'd change the name here. Maybe `USE_IMPORT_FOUNDATION` or `USE_IMPORT_SWIFTFOUNDATION`.